migration/v0.38/prepare/60-adopt-dns-records.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+ROOT="$(readlink -f "${HERE}/../../../")"
+
+# shellcheck source=scripts/migration/lib.sh
+source "${ROOT}/scripts/migration/lib.sh"
+
+log_info "Adopt DNS records for external-dns"
+
+if yq_check common .externalDns.enabled true; then
```

Comment:
@anders-elastisys I can add a check if it is enabled in any sc/wc/common. But how I adopt dns records at the moment does not really mater which cluster you pick. As I create new TXT records for all records that exists in aws and not in the clusters. As that it much easier and for any record that external-dns does not watch will not have any affect.

So I do not see the need of adding more sc/wc specific checks. 
Or do you want me to check what dns records will be watched by external-dns per cluster and then do the adoption?

Reply:
I think just checking if it is enabled or not in each separate config file at the start should be fine with what you are saying. I guess this migration script only supports if service type loadbalancer is supported by the cloud provider, or how should it work if you manually add endpoints to the externalDns config?

Reply:
It should be with any of the supported types, CRD, ingress and service. As external-dns will not touch dns records if the owner txt record is missing. 

Also it will just look at records in the specified `domains/baseDomain`.